pkg/clusternetwork/config.go

```diff
@@ -0,0 +1,77 @@
+package clusternetwork
+
+import (
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	operatorv1 "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterNetworkConfig interface contains methods to validate network configuration of a cluster
+type ClusterNetworkConfig interface {
+	Validate() error
+}
+
+// networkType information for a required network type
+type networkType struct {
+	// name describes value of the Network Type
+	name string
+	// clienset is the OpenShift config client, we will use to query the OpenShift API object status
```

Comment:
Incorrect doc. While fixing this please make it `ocClient`. 

`...we will use to interact with the OpenShift API`

Reply:
Done